Source/santad/SNTSyncdQueue.m

```diff
@@ -38,33 +38,30 @@ - (instancetype)init {
   return self;
 }
 
-- (void)addBundleEvents:(NSArray<SNTStoredEvent *> *)events {
-  if (![self backoffForBundleHash:events.firstObject.fileBundleHash]) return;
+- (void)addEvents:(NSArray<SNTStoredEvent *> *)events isFromBundle:(BOOL)isFromBundle {
+  if (!events || events.count == 0) return;
```

Comment:
nit: if (!events || !events.count) return;

Reply:
fixed

Reply:
You can avoid the first condition altogether

Reply:
oops, done.